src/RestApi/StoreApi/Schemas/CartSchema.php

```diff
@@ -54,106 +39,117 @@ protected function get_properties() {
-				'description' => __( 'Lines of taxes applied to items and shipping.', 'woo-gutenberg-products-block' ),
-				'type'        => 'array',
-				'context'     => [ 'view', 'edit' ],
-				'readonly'    => true,
-				'items'       => [
-					'type'       => 'object',
-					'properties' => [
-						'name'  => [
-							'description' => __( 'The name of the tax.', 'woo-gutenberg-products-block' ),
+				'properties'  => array_merge(
+					$this->get_store_currency_properties(),
+					[
+						'total_items'        => [
+							'description' => __( 'Total price of items in the cart.', 'woo-gutenberg-products-block' ),
 							'type'        => 'string',
 							'context'     => [ 'view', 'edit' ],
 							'readonly'    => true,
 						],
-						'price' => [
-							'description' => __( 'The amount of tax charged. Amount provided using the smallest unit of the currency.', 'woo-gutenberg-products-block' ),
+						'total_items_tax'    => [
+							'description' => __( 'Total tax on items in the cart.', 'woo-gutenberg-products-block' ),
 							'type'        => 'string',
 							'context'     => [ 'view', 'edit' ],
 							'readonly'    => true,
 						],
-					],
-				],
+						'total_fees'         => [
+							'description' => __( 'Total price of any applied fees.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_fees_tax'     => [
+							'description' => __( 'Total tax on fees.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_discount'     => [
+							'description' => __( 'Total discount from applied coupons.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_discount_tax' => [
+							'description' => __( 'Total tax removed due to discount from applied coupons.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_shipping'     => [
+							'description' => __( 'Total price of shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_shipping_tax' => [
+							'description' => __( 'Total tax on shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_tax'          => [
+							'description' => __( 'Total tax applied to items and shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_price'        => [
```

Comment:
Is this a common name used for this value throughout woo and it's api?  Would this be equivalent to `grand_total` and would that be a clearer term to use?

Reply:
Urgh looks like my docs updates were removed after rebase. fml. Will resubmit.

`grand_total` is not used anywhere in core either. `total_price` follows the pattern of the other totals defined here.

Reply:
Actually no, I was in wrong branch. The inline docs have been updated - see the readme.md file. @nerrad 

`line_subtotal` is not equivalent because it's for the line, not a single item. But it is all we need. Product price can come from products API.

Reply:
> Product price can come from products API.

does that mean an additional request though (or would that be necessary anyways to get the product details for the items in the cart)?

Reply:
@nerrad yes it would, however, it's not needed for the cart and checkout :) Cart and checkout only deals with totals, even for displaying items.